### PR TITLE
Add developer docs for lighting functionality

### PR DESCRIPTION
**Pull request by georgios-uber**

### CODE REVIEW

File: docs/developer-guide/lighting.md [hunk 1]
```diff
@@ -0,0 +1,42 @@
+# Lighting
+
+Luma.gl supports different shading modules.
+To make
```
Comment (jianhuang01) [inline docs/developer-guide/lighting.md]: to make what?

File: docs/developer-guide/lighting.md [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+
+
+## Light Sources
+
+## Example
+You can use the `lighting` example to play with lights.
+
+## Currently supported lighting models
+- Phong (Available in the `phong-lighting` module)
+- PBR (Available in the `pbr` module)
```
Comment (jianhuang01) [inline docs/developer-guide/lighting.md]: better to link the Phong and PBR module folder or code file, and remove the comment in ()

File: docs/developer-guide/lighting.md [hunk 3]
```diff
@@ -0,0 +1,42 @@
+# Lighting
+
+Luma.gl supports different shading modules.
+To make 
+
+
+## Light Sources
+
+## Example
+You can use the `lighting` example to play with lights.
```
Comment (jianhuang01) [inline docs/developer-guide/lighting.md]: better to change `lighting` to [lighting](link to example)

File: docs/developer-guide/lighting.md [hunk 4]
```diff
@@ -0,0 +1,42 @@
+# Lighting
+
+Luma.gl supports different shading modules.
+To make 
+
+
+## Light Sources
```
Comment (ibgreen) [inline docs/developer-guide/lighting.md]: Give JS code examples of how to create a model with a material and set lighting uniforms... explain that we have two Material systems that both support the same lights...

**Comment by ibgreen** [comment]
Closing, feel free to reopen if comments addressed.

**Pull request closed without merging**